mathesar/exception_handlers.py

```diff
@@ -80,6 +81,7 @@ def mathesar_exception_handler(exc, context):
                 response_data['code'] = error_code
                 response_data['message'] = error_message
                 response_data['details'] = {'exception': force_str(exc)}
+                response_data['stacktrace'] = reformat_stacktrace(traceback.format_exc())
```

Comment:
We should also be able to show the stacktrace even for captured errors that are properly formatted(pretty errors).  Please take a look at line https://github.com/centerofci/mathesar/blob/34d6eb3882c47ff4f43ff346e85c3eac6f0c07e2/mathesar/exception_handlers.py#L62

The response body of those errors should also contain the stacktrace.

Reply:
Thanks, I'll get on it.